types/concurrency-limiter/test/concurrency-limiter-tests.ts

```diff
@@ -0,0 +1,20 @@
+import Limiter = require("concurrency-limiter");
+
+// @ts-expect-error
+new Limiter();
+new Limiter(2);
+
+// @ts-expect-error
+new Limiter(2)._limit;
+// @ts-expect-error
+new Limiter(2)._length;
+// @ts-expect-error
+new Limiter(2)._queue;
+
```

Comment:
those are not required (check earlier comment).
If you don't use as indexed access types, there is no point of adding declaration for private fields